### PR TITLE
Specify GH project board per team

### PR DESCRIPTION
**Pull request by alxnddr**
### Description

Previously we all used a single board https://github.com/orgs/metabase/projects/50/views/1 but not only two team use it. In order not to pollute their board with PRs from other teams, now every team will specify their own board if needed.

### How to verify

Check the changes are sensible.
Example run where it did not add this PR to the board: https://github.com/metabase/metabase/actions/runs/8160443973/job/22307033268

### CODE REVIEW

File: .github/team.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     {
       "name": "Querying Components",
       "label": ".Team/QueryingComponents",
+      "projectUrl": "https://github.com/orgs/metabase/projects/50/views/1",
```
Comment (nemanjaglumac) [inline .github/team.json:6]:
```suggestion
      "projectUrl": "https://github.com/orgs/metabase/projects/74",
```

File: .github/team.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
     {
       "name": "Querying Components",
       "label": ".Team/QueryingComponents",
+      "projectUrl": "https://github.com/orgs/metabase/projects/74",
```
Comment (alxnddr) [inline .github/team.json:6]: Hey @metabase/querying-components, would you like PRs you create get assigned to the board?

File: .github/team.json (hunk 1 above)
Comment (alxnddr) [reply .github/team.json]: Good catch, thanks. I wanted to add this url to the QP team. And @NevRA just told me they don't need PR auto-assignment

**Review by darksciencebase — APPROVED** [review]
thank you so much.

**Pull request merged**